assets/src/edit-story/components/checklist/constants.js

```diff
@@ -277,7 +277,7 @@ export const DESIGN_COPY = {
       `${MIN_VIDEO_RESOLUTION}p`
     ),
   },
-  videoResolutionTooHigh: {
```

Comment:
This was simply incorrect as the message was never about video _resolution_ but video _duration_.